src/Analysis/GWASResults/Views/Input/AttritionTable/AttrtitionTable.jsx

```diff
@@ -19,6 +68,23 @@ const AttritionTable = ({ tableData, title }) => {
     return <h3>❌</h3>;
   };
 
+  const displayNumberOrX = (data) => {
+    if(data || data == 0)
+    {
+      return data;
+    }
+    return <h3>❌</h3>;
+  }
+
+  let hareGroupNames = defaultHareGroups;
+
+  if(tableData.rows)
+  {
+    console.log('start to get hare group names')
```

Comment:
Please remove console.log statements, it is generally considered a best practice to keep these out of production unless there is a good reason to include them.